siriuspy/tests/ramp/test_waveform.py

```diff
@@ -222,29 +218,27 @@ def test_update(self):
 class TestWaveform(TestCase):
     """Test Waveform class."""
 
-    PUBLIC_INTERFACE = (
+    PUB_INTERFACE = (
         'duration',
         'update',
     )
 
     def test_public_interface(self):
         """Test module's public interface."""
         valid = util.check_public_interface_namespace(
-                Waveform,
-                TestWaveform.PUBLIC_INTERFACE)
+            Waveform, TestWaveform.PUB_INTERFACE)
         self.assertTrue(valid)
 
     def test_duration(self):
         """Test duration."""
         # TODO: implement it!
 
-
     def _test_update(self):
         """Test update."""
         # TODO: implement using mock to substitute server
         bo_b = WaveformDipole()
-        defoc = Waveform(maname='BO-Fam:MA-QD', dipole=bo_b)
+        defoc = Waveform(psname='BO-Fam:MA-QD', dipole=bo_b)
```

Comment:
@fernandohds564 , sure about this change?

Reply:
@anacso17 ?

Reply:
This is a right fix!

Reply:
@anacso17 , it was not that. I found strange that the name of the `Waveform` constructor argument is `psname` and the string passed is of _MA_ discipline. I guess this a side effect of abandoning MA IOCs. We will eventually finalize refactoring of classes that used _ maname_...

Reply:
@anacso17 , after you read my response, can you please resolve this conversation?

Reply:
Oh, my mistake, the discipline is really wrong!